app/plugins/aws-lambda.md

```diff
@@ -128,7 +128,7 @@ When the plugin is added to an API entity (which is deprecated as of 0.13.0),
 it is the `upsream_url` property which must be specified and resolvable as well
 (but ignored).
 
-#### Response plugins
+**Response** plugins
```

Comment:
This update needs to be made in the new `app/_hub/kong-inc/aws-lambda/index.md` file.

Reply:
Actually #### Response plugins should be fine - I will actually reverse this one